CMakeLists.txt

```diff
@@ -50,10 +50,16 @@ set(CMAKE_CXX_EXTENSIONS OFF)
 
 if(ENABLE_LIBCXX)
     add_compile_options($<$<COMPILE_LANG_AND_ID:CXX,Clang>:-stdlib=libc++>)
-    add_link_options(
-        $<$<LINK_LANG_AND_ID:CXX,Clang>:-lc++>
-        $<$<LINK_LANG_AND_ID:CXX,Clang>:-lc++abi>
-    )
+    #add_link_options(
+    #    $<$<LINK_LANG_AND_ID:CXX,Clang>:-lc++>
+    #    $<$<LINK_LANG_AND_ID:CXX,Clang>:-lc++abi>
+    #)
+endif()
+
+# Using below until we can move to CMake >= 3.18 for LINK_LANG_AND_ID
```

Comment:
❤️

Reply:
Sorry for leading you astray.  Didn't look at the version and assumed that was an old one.